engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -168,8 +168,12 @@ object WorkflowExecutionActor {
         case _ => Nil
       }
 
+      def isDone(e: JobKey): Boolean = executionStore.store exists {
+        case (k, s) => k.scope == e.scope && k.index == e.index && s == ExecutionStatus.Done
```

Comment:
To be clear, it's been changed to `case (k, s) => k.scope.fullyQualifiedName == e.scope.fullyQualifiedName && k.index == e.index && s == ExecutionStatus.Done`